fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/api/FedoraToOCFLObjectIndex.java

```diff
@@ -29,16 +29,28 @@
      * Retrieve identification information for the OCFL object which either contains, or is identified by,
      * the provided fedora resource id. In other words the method will find the closest resource that is persisted
      * as an OCFL object and returns its identifiers.
-     *
+     * <p>
```

Comment:
This is an IntelliJ thing, which you can disable.
`Preferences -> Editor -> Code Style -> Java -> JavaDoc -> Other -> Generate "<p>" on empty lines`

Reply:
👍